data/forms/city/weekly_funding.form

```diff
@@ -49,17 +55,29 @@
       </label>
       <label id="FUNDING_ADJUSTMENT">
         <position x="52" y="212"/>
-        <size width="400" height="15"/>
+        <size width="0" height="15"/>
+        <alignment horizontal="left" vertical="centre"/>
+        <font>smalfont</font>
+      </label>
+      <label id="FUNDING_ADJUSTMENT_VALUE">
+        <position x="0" y="0"/>
+        <size width="0" height="15"/>
         <alignment horizontal="left" vertical="centre"/>
         <font>smalfont</font>
       </label>
       <label id="FUNDING_NEW">
         <position x="52" y="240"/>
-        <size width="400" height="15"/>
+        <size width="0" height="15"/>
         <alignment horizontal="left" vertical="centre"/>
         <font>smalfont</font>
       </label>
-
+      <label id="FUNDING_NEW_VALUE">
+        <position x="0" y="0"/>
```

Comment:
Having an empty position doest seem right here, the goal of the form xml files is that they can be altered to reskin the game, and so just setting the position in code, instead of using a value relative to an anchor in the xml, limits that.

Possibly the same with the zero sized stuff above, I don't think this screen can reflow (ie each line has a set contents and can't flow to a new line and push everything else down anyway, so having a zero size just makes it less clear to whoever's reading the xml expecting it to relate to the final output)

Reply:
Agree. It's not the best solution. 
What do you suggest?